src/warc2zim/content_rewriting/html.py

```diff
@@ -67,16 +94,31 @@ def handle_starttag(self, tag: str, attrs: AttrsList, *, auto_close: bool = Fals
                 if preload_type == "script":
                     self.html_rewrite_context = "js-classic"
 
+        # Handle special case of <base> tag which have to be simplified (remove href)
+        # and hence write only if not empty
+        if tag == "base":
+            values = " ".join(
+                self.format_attr(*attr)
+                for attr in [
+                    (attr_name, attr_value)
+                    for (attr_name, attr_value) in attrs
+                    if attr_name != "href"
```

Comment:
Because `attrs` is not a dictionary but at list of tuple (name, value) 

I would have preferred a dictionary, but I assume the list is mandatory because theoretically we can probably have multiple times the same attr name in a given tag - even if it probably does not fit the HTML spec ; I preferred to not changed this at least.